src/components/Badge/Badge.tsx

```diff
@@ -3,17 +3,21 @@ import classNames from 'classnames';
 
 import './Badge.scss';
 
+export const TRENDING_THEME = 'trending';
+export const BESTSELLER_THEME = 'bestseller';
+
 const THEMES = {
-  bestseller: 'Bestseller',
-  trending: 'Trending',
+  [BESTSELLER_THEME]: 'Bestseller',
+  [TRENDING_THEME]: 'Trending',
   'fall-essential': 'Fall Essential'
```

Comment:
why not also `fall-essential`?

Reply:
because i didnt need it elsewhere, but sure I can do it

Reply:
LGTM otherwise